WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverViewController.swift

```diff
@@ -2,71 +2,55 @@ import UIKit
 import SwiftUI
 import WordPressUI
 
-final class CompliancePopoverViewController: UIViewController {
+final class CompliancePopoverViewController: UIHostingController<CompliancePopover> {
 
     // MARK: - Dependencies
 
     private let viewModel: CompliancePopoverViewModel
-
-    // MARK: - Views
-    private let hostingController: UIHostingController<CompliancePopover>
-
-    private var contentView: UIView {
-        return hostingController.view
-    }
-
     private var bannerIntrinsicHeight: CGFloat = 0
 
     init(viewModel: CompliancePopoverViewModel) {
         self.viewModel = viewModel
-        hostingController = UIHostingController(rootView: CompliancePopover(viewModel: self.viewModel))
-        super.init(nibName: nil, bundle: nil)
+        super.init(rootView: CompliancePopover(viewModel: viewModel))
     }
 
     required dynamic init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    // MARK: - View Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.addContentView()
-        hostingController.view.translatesAutoresizingMaskIntoConstraints = true
-        hostingController.rootView.goToSettingsAction = {
+        rootView.goToSettingsAction = {
             self.viewModel.didTapSettings()
         }
-        hostingController.rootView.saveAction = {
+        rootView.saveAction = {
             self.viewModel.didTapSave()
         }
         viewModel.didDisplayPopover()
     }
 
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
-        let targetSize = CGSize(width: view.bounds.width, height: 0)
-        let contentViewSize = contentView.systemLayoutSizeFitting(targetSize)
-        self.contentView.frame = .init(origin: .zero, size: contentViewSize)
-        self.preferredContentSize = contentView.bounds.size
-
-        self.hostingController.rootView.screenHeight = self.view.frame.height
```

Comment:
@alpavanoglu The `screenHeight` property is not used anymore. Maybe it should be removed.